website/content/projects/2019/spec.md

```diff
@@ -118,11 +118,62 @@ The project will span seven weeks (Weeks 4 - 10), with a final review and team p
 *   Weeks 9-10: Sprint 4
 
 
-<!--
+
 ## Suggested initial goals
 
-TODO
--->
+To get you started, we suggest each team focus on the following set of initial goals, most of which won't require much communication with other teams at first (allowing you to focus on the requirements of your component before you have to start collaborating with other teams)
+
+### WDL
+
+We don't expect you to define a language completely from scratch, and suggest that you use [YAML](https://en.wikipedia.org/wiki/YAML) as the base format for the WDL. We have even provided some sample WDL files to get you started. However, you shouldn't regard these files as normative: they are a suggestion on what a WDL file would look like, but it is up to you to decide the exact format of the files. Furthermore, while YAML files can be parsed in C using the [libyaml](https://github.com/yaml/libyaml) library, we have provided a library called libobj that provides an abstraction over the low-level operations provided by libyaml.
+
+So, the WDL team can initially focus on this:
+
+- Writing a draft specification of the WDL language
+- Writing a module for loading WDL files. This module must do some basic validation of the file (ensuring it meets the specification of the WDL language)
```

Comment:
What will they be loading the files into? `libobj` objects?

Reply:
As a first step, yet. However, they ultimately have to load them into data structures produced by Game State

Reply:
Ok cool, we should specify that so that they don't get confused and think they're blocked on game state

Reply:
Pushed a fix to this.